src/components/UploadingJobsDisplay/components/FileItem/FileItem.js

```diff
@@ -3,27 +3,30 @@ import React, {
   Fragment,
 } from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
 import classNames from 'classnames';
+import noop from 'lodash/noop';
 
 import {
   Icon,
   IconButton,
 } from '@folio/stripes/components';
 
-import {
-  UPLOADED,
-  UPLOADING,
-  FAILED,
-} from './fileItemStatuses';
+import * as fileStatuses from './fileItemStatuses';
 import Progress from '../../../Progress';
 
 import css from './FileItem.css';
 
-const getFileItemMeta = (props) => {
+const getFileItemMeta = props => {
```

Comment:
I suggest to move getFileItemMeta from the file to another place, and then import it and use in the component

Reply:
Makes sense. Will do.